sos/report/plugins/systemd.py

```diff
@@ -39,6 +39,7 @@ def setup(self):
             # status --all mostly seems to cover the others.
             "systemctl list-units",
             "systemctl list-units --failed",
+            "systemctl list-units --all",
             "systemctl list-unit-files",
```

Comment:
A question from `systemctl` noob: what *additional* information the `list-units --all` provide, that is not covered in the three other command outputs? (we try to limit redundant duplicate data collection, if possible)